manager/garden_manager.py

```diff
@@ -59,7 +64,62 @@ def find_all_with_orchard(self) -> list:
         """
         return [garden for garden in self.gardens if garden.has_orchard()]
 
-    def __str__(self):
+    def get_lsit_of_has_orchard(self) -> list[bool]:
+        """
+        Returns a list of boolean values indicating whether each garden in manager has an orchard.
+
+        Returns:
+            list[bool]: A list of boolean values.
+                        indicating whether the garden has an orchard (True) or not (False).
+        """
+        return [garden.has_orchard() for garden in self.gardens]
+
+    def get_enumerate(self):
+        """
+        Get the enumerated list of gardens.
+
+        Returns:
+            list: A list of tuples containing the index and garden object.
+        """
+        return list(enumerate(self.gardens))
+
+    def get_zip(self):
+        """
+        Get a zipped list of orchard status and gardens.
+
+        Returns:
+            list: A list of tuples containing the orchard status and corresponding garden object.
+        """
+        return list(zip(self.get_lsit_of_has_orchard(), self.gardens))
+
+    def is_all_have_orchard(self) -> bool:
+        """
+        Check if all gardens have an orchard.
+
+        Returns:
+            bool: True if all gardens have an orchard, False otherwise.
+        """
+        return all(self.get_lsit_of_has_orchard())
+
+    def is_at_least_one_have_orchard(self) -> bool:
+        """
+        Check if at least one garden has an orchard.
+
+        Returns:
+            bool: True if at least one garden has an orchard, False otherwise.
+        """
+        return any(self.get_lsit_of_has_orchard())
```

Comment:
Task states that you need to make one method to return with this signature: {"any": ..., "all": ...}. But I guess it's alright